packages/elastic-analytics/src/events/types.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export interface EventContext {
+  // TODO: Extend with known keys
```

Comment:
This TODO will be done in upcoming PRs when we come to an agreement on the minimum piece of context we'd like to provide